### PR TITLE
Get list block

### PR DESCRIPTION
**Pull request by ajpal**
This is a prototype of the getList block to unblock curriculum on writing lessons.
image
Right now the first dropdown opens the asset manager (will eventually be the dataset picker) and the second dropdown shows true/false (will eventually be auto-populated with the columns of the selected dataset).
The getList function works synchronously to read the entire table, and return a single list containing the values from the selected column. 
Once the dataset picker is implemented, it will import the selected dataset, but in the meantime, to use getList, you must first import your desired table in the data tab.

### CODE REVIEW

File: apps/src/applab/commands.js [hunk 1]
```diff
@@ -1789,6 +1789,20 @@ applabCommands.handleReadValue = function(opts, value) {
   }
 };
 
+applabCommands.getList = function(opts) {
+  var onSuccess = handleGetListSync.bind(this, opts);
+  var onError = handleGetListSyncError.bind(this, opts);
+  Applab.storage.readRecords(opts.tableName, {}, onSuccess, onError);
+};
+var handleGetListSync = function(opts, values) {
+  let l = [];
```
Comment (jmkulwik) [inline apps/src/applab/commands.js]: More descriptive name? (same for x below)

Comment (ajpal) [reply apps/src/applab/commands.js]: whoops, sorry- missed cleaning this up

Comment (jmkulwik) [reply apps/src/applab/commands.js]: [nit] and newlines between function definitions

**Review by jmkulwik — APPROVED** [review]
Prototype looks good. Left a couple small comments.

**Pull request merged**